judge/judgedaemon.main.php

```diff
@@ -538,6 +538,14 @@ function fetch_executable_internal(
     exit(1);
 }
 
+if (defined('STDLOG')) {
+    // Add a lock to the logfile; this makes sure we can only run the daemon once
+    if (!flock(STDLOG, LOCK_EX | LOCK_NB)) {
+        echo "Can lock logfile, is a daemon already running?\n";
```

Comment:
If you can, should you exit?

Reply:
Can't*. But @eldering proposes to use a separate lockfile and he will implement that.

Reply:
Also I'd rather not change this variable name now, its pretty old.

Reply:
Please also log the path to the lockfile

Reply:
@eldering will use a separate file and I guess he can add the logmsg for the filename 😄